app/connectors/ApplicationConnector.scala

```diff
@@ -244,10 +244,10 @@ abstract class ApplicationConnector(implicit val ec: ExecutionContext) extends R
     encode(str, encoding)
   }
 
-  def searchCollaborators(apiContext: ApiContext, apiVersion: String, partialEmailMatch: Option[String])(implicit hc: HeaderCarrier): Future[Seq[String]] = {
+  def searchCollaborators(apiContext: ApiContext, apiVersion: ApiVersion, partialEmailMatch: Option[String])(implicit hc: HeaderCarrier): Future[Seq[String]] = {
     val queryParameters = List(
       "context" -> apiContext.value,
-      "version" -> apiVersion
+      "version" -> apiVersion.value
```

Comment:
I wonder if these need urlEncoding.

Reply:
They weren't urlEncoded before so I left as was

Reply:
That means nothing ;-)